doozerlib/dblib.py

```diff
@@ -123,6 +123,24 @@ def check_missing_db_env_var(self):
         self.runtime.logger.info("Found all environment variables required for db setup.")
         return True
 
+    def select(self, expr, limit):
```

Comment:
could this have a comment? just looking at this naively, i'd want to know:

1. why do we need this at all
2. what's the format of `expr`
3. why not have a default `limit=100` or something